alert/blueprints/application/routes.py

```diff
@@ -1,12 +1,10 @@
 from flask import Blueprint, render_template
-from flask_babel import _
```

Comment:
Don't think this is needed at the moment. Can't see localization being an immediate requirement, so let's come back to this later. Remove the import for now

Reply:
👍